view/templates/field/range_percent.tpl

```diff
@@ -0,0 +1,20 @@
+<div id="id_{{$field.0}}_wrapper" class="form-group field range">
+{{if !isset($label) || $label != false }}
```

Comment:
`isset` is working in templates?

Reply:
Smarty templates are converted to PHP files, so many PHP functions work as is in templates as well.